### PR TITLE
some tidying for a release of 1.8.0

### PR DESCRIPTION
**Pull request by leej3**
Note: It needs MFC classes on win64 to build (otherwise it is missing an afxres.h header file)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,6 +10,10 @@ source:
   patches:
     - patches/fix-jpeg-boolean-usage.patch
 
+build:
+  ignore_run_exports:
+  skip: true # [win32]
+    - zlib
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: To fix build section:
```
build:
  number: 0
  skip: true  # [win32]
  ignore_run_exports:
    - zlib
```

**Review by varlackc — COMMENTED** [review]
We probably should use `https` instead of `http`
https://github.com/AnacondaRecipes/devil-feedstock/blob/update_for_release/recipe/meta.yaml#L8

https://github.com/AnacondaRecipes/devil-feedstock/blob/update_for_release/recipe/meta.yaml#L41

https://github.com/AnacondaRecipes/devil-feedstock/blob/update_for_release/recipe/meta.yaml#L46

probably migth have to find if the correct dev ur is in github, not sure where the upstream source code is located

https://sourceforge.net/projects/openil/#:~:text=Where%20is%20the%20github%20repo%3F

**Comment by leej3** [comment]
[This](https://github.com/DentonW/DevIL/archive/refs/tags/v1.8.0.tar.gz) would be the alternative for the source...

EDIT: those links don't work for me over https...

File: recipe/meta.yaml (hunk 1 above)
Comment (leej3) [reply recipe/meta.yaml]: thanks

**Comment by varlackc** [comment]
> [This](https://github.com/DentonW/DevIL/archive/refs/tags/v1.8.0.tar.gz) would be the alternative for the source...
> 
> EDIT: those links don't work for me over https...

this link might work as a source for development, since the upstream source seems to be pointing to some existing code.

https://sourceforge.net/projects/openil/files/DevIL/1.8.0/

also the github source could work as well, however we tend to prefer `github` urls for `dev_source`
https://github.com/DentonW/DevIL


The following link seem to open properly

https://downloads.sourceforge.net/openil/DevIL-1.8.0.tar.gz

in this particular case we should change the `source url` to use `https` instead of `http` as following

url: https://downloads.sourceforge.net/openil/DevIL-{{ version }}.tar.gz

the `home_url` and `doc_url` do not seem to support `https` at this time so probably we should live those as `http` for now.

**Review by skupr-anaconda — COMMENTED** [review]
The license is `LGPL-2.1-or-later` and `license_family` is `LGPL`

**Comment by skupr-anaconda** [comment]
An error on s390x
```
conda.exceptions.ResolvePackageNotFound:
  - jasper
```

We can skip building on s390x

**Comment by leej3** [comment]
Thanks, skipped.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**